### DISCUSSION TITLE
Live m3u8 streams - player Svelte

### DISCUSSION THREAD

**Discussion by marksteven** [sveltejs/kit#9783] [Q&A]
### Describe the problem

I am not able to play a live stream (type m3u8 ) through a sveltekit site. Happily place a mp4 file and play using the various media examples, but to date have not found a svelte solution for streaming video.

### Describe the proposed solution

HLS or similar addition to svelte for simple working example player.- That works with current sveltekit 

### Alternatives considered

Have tried many jwplayer, vime, React-hls and others.
Vidstack is looking promising but again could not get it to work (for me)

### Importance

i cannot use SvelteKit without it

### Additional Information

At this stage cannot use sveltekit till I can provide a streaming solution player, no matter how simple. So will have to keep using React till I can find a solution.

Any suggestions out there on a solution gratefully accepted.

**Comment by dennisregalado** [comment]
What troubles did you have when using HLS? [Swehl](https://swehl.com/) streams all video using [mux/video](https://github.com/muxinc/elements/tree/main/packages/mux-video)

**Reply by marksteven** [reply]
Using hls.js and a few other variants did not work in svelte. Did not get errors in console but video did not play, simply had a blank screen. Thankyou for the reference link as it is one I have not tried. Will give it a go. Thankyou
